pkg/crc/machine/start.go

```diff
@@ -169,8 +169,8 @@ func runGrowpart(sshRunner *crcssh.Runner, rootPart string) error {
 	return nil
 }
 
-func growLVForMicroshift(sshRunner *crcssh.Runner) (string, error) {
-	rootPart, _, err := sshRunner.RunPrivileged("Get PV disk path", "/usr/sbin/pvs", "-S", "lv_name=root", "-o", "pv_name", "--noheadings")
+func growLVForMicroshift(sshRunner *crcssh.Runner, lvFullName string) (string, error) {
```

Comment:
By passing `lvFullName` as an arg, you no longer need to return `lvPath` from this method, which simplifies the other commit. I'd start the PR with this refactor, and then the commit adding the workaround for the resize.

Reply:
@cfergeau so what you want is from the caller something like following?
```
if preset == crcPreset.Microshift {
               lvFullName := "rhel/root"
               rootPart = fmt.Sprintf("/dev/%s", lvFullName)
		err = growLVForMicroshift(sshRunner, lvFullName)
		if err != nil {
			return err
		}
	}
```

Reply:
Yes unless you think it does not make sense? This would be close to the way the other codepath is structured.

Reply:
Alright, I will make that change, wanted to make sure I understood correctly.

Reply:
@cfergeau updated.

Reply:
Thanks  for the refactor!